frontend/src/components/pages/HomePage.tsx

```diff
@@ -4,7 +4,7 @@ import Filter from "../homepage/Filter";
 import StoryList from "../homepage/StoryList";
 import { StoryCardProps } from "../homepage/StoryCard";
 import AuthContext from "../../contexts/AuthContext";
-import ToggleButton from "../navigation/ToggleButton";
+import ToggleButton from "../utils/ToggleButton";
```

Comment:
> Is there another name for "generic UI components" that we could throw ToggleButton into?

Good question! There is the [chakra ui switch](https://chakra-ui.com/docs/form/switch), but it's design needs a total overhaul
![image](https://user-images.githubusercontent.com/15113249/126411368-737b2c09-ae6a-4c46-828d-0b54b15baf3f.png)

> Curious what other people think and maybe this is a normal practice I'm not aware of, but the double /utils folders with fairly different purposes is a bit trippy

I'm personally fine with it since the file is in `/components` in this case, but open to suggestions!